WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -21,70 +21,70 @@ enum WooConstants {
+    /// List of thrusted URLs
     ///
-    static var privacyURL: URL {
-        trustedURL("https://automattic.com/privacy/")
-    }
+    enum URLs: String, CaseIterable {
 
-    /// Privacy policy for California users URL
-    ///
-    static var californiaPrivacyURL: URL {
-        trustedURL("https://automattic.com/privacy/#california-consumer-privacy-act-ccpa")
-    }
+        /// Jetpack Setup URL
+        ///
+        case jetpackSetup = "https://jetpack.com/support/getting-started-with-jetpack/"
 
-    /// Help Center URL
-    ///
-    static var helpCenterURL: URL {
-        trustedURL("https://docs.woocommerce.com/document/woocommerce-ios/")
-    }
+        /// Terms of Service Website. Displayed by the Authenticator (when / if needed).
+        ///
+        case termsOfService = "https://wordpress.com/tos/"
 
-    /// Feature Request URL
-    ///
-    static var featureRequestURL: URL {
-        trustedURL("http://ideas.woocommerce.com/forums/133476-woocommerce?category_id=84283")
-    }
+        /// Cookie policy URL
+        ///
+        case cookie = "https://automattic.com/cookies/"
 
-    /// URL used for Learn More button in Orders empty state.
-    ///
-    static var blogURL: URL {
-        trustedURL("https://woocommerce.com/blog/")
-    }
+        /// Privacy policy URL
+        ///
+        case privacy = "https://automattic.com/privacy/"
 
-    /// URL for in-app feedback survey
-    ///
-    static var inAppFeedbackURL: URL {
-        if BuildConfiguration.current == .localDeveloper {
-            return trustedURL("https://wasseryi.survey.fm/woo-mobile-app-test-survey")
-        } else {
-            return trustedURL("https://automattic.survey.fm/woo-app-general-feedback-user-survey")
-        }
-    }
+        /// Privacy policy for California users URL
+        ///
+        case californiaPrivacy = "https://automattic.com/privacy/#california-consumer-privacy-act-ccpa"
 
-    /// Number of section events required before an app review prompt appears
-    ///
-    static let notificationEventCount = 5
+        /// Help Center URL
+        ///
+        case helpCenter = "https://docs.woocommerce.com/document/woocommerce-ios/"
 
-    /// Number of system-wide significant events required
-    /// before an app review prompt appears
-    ///
-    static let systemEventCount = 10
+        /// Feature Request URL
+        ///
+        case featureRequest = "http://ideas.woocommerce.com/forums/133476-woocommerce?category_id=84283"
+
+        /// URL used for Learn More button in Orders empty state.
+        ///
+        case blog = "https://woocommerce.com/blog/"
+
+        /// URL for in-app feedback survey
+        ///
+#if DEBUG
+        case inAppFeedback = "https://wasseryi.survey.fm/woo-mobile-app-test-survey"
+#else
+        case inAppFeedback = "https://automattic.survey.fm/woo-app-general-feedback-user-survey"
+#endif
+
+        /// Returns the URL version of the receiver
+        ///
+        func asURL() -> URL {
```

Comment:
This bugs me a bit because the `enum` name is `URLs` but it's actually a `String` enumeration. But I guess there's no other option, right?

Reply:
I was also struggling with this name but at the end I noticed that making it something more technically close to the raw type made it feel less semantically cohesive in when using it.

I'm open for suggestions tho! :-)

Reply:
Yeah, I tried `enum URLs: URL`. It didn't work well. 😄 But I really like the automated unit test! I wonder if we can do something similar for the [`IconsTests`](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/WooCommerceTests/Extensions/IconsTests.swift).